Misc/NEWS.d/next/Library/2021-08-10-00-05-53.bpo-44859.9e9_3V.rst

```diff
@@ -0,0 +1,7 @@
+Raise more accurate exceptions in :mod:`sqlite3`.
+
+* Raise InterfaceError iso. ProgrammingError for SQLITE_MISUSE
```

Comment:
```suggestion
* Raise InterfaceError instead of ProgrammingError for SQLITE_MISUSE
```
I don't think "iso." is a widely understood abbreviation.

Reply:
Yes; it's a bad habit I have. Thanks for catching.